index.html

```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
       </p>
       <section>
         <h3>
+          Privacy and security considerations
+        </h3>
+        <p>
+          When the <a>fullscreen</a> <a>display mode</a> is applied, it is
+          RECOMMENDED that the user agent still afford the end-user a means to
+          access/view the origin, or URL, and icon of the Web application. This
```

Comment:
This presupposes access to the OS's task switcher - which, for instance, wouldn't be true for Firefox on Android. I'm just going to say that it's left up to the implementer to come up with something.